lib/parity/backup.rb

```diff
@@ -34,21 +34,32 @@ def restore_from_development
     end
 
     def restore_to_development
-      drop_development_database
-      pull_remote_database_to_development
+      download_remote_backup
+      wipe_development_database
+      restore_from_local_temp_backup
+      delete_local_temp_backup
     end
 
-    def drop_development_database
-      Kernel.system("dropdb #{development_db}")
+    def wipe_development_database
+      Kernel.system("dropdb #{development_db} && createdb #{development_db}")
     end
 
-    def pull_remote_database_to_development
+    def download_remote_backup
       Kernel.system(
-        "heroku pg:pull DATABASE_URL #{development_db} --remote #{from} "\
-          "#{additional_args}",
+        "curl -o tmp/latest.backup $(#{from} pg:backups public-url -q)",
```

Comment:
I think the `$(#{from} ...)` should be quoted, as there's no reason the URL might not contain, e.g., `&` or other nasty shell characters.